### PR TITLE
Extract modal alignment related inline styles and transfer them into …

### PR DESCRIPTION
**Pull request by beroes**
…the stylesheet

https://redbooth.com/a/#!/mytasks/20835109

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "iptools-jquery-modal",
   "description": "Multifunctional jQuery modal component",
   "main": "dist/iptools-jquery-modal.min.js",
-  "version": "0.1.3",
+  "version": "0.2.1",
```
Comment (ain) [inline bower.json]: @beroes I think I mixed it up with mlhmenu by mentioning the version could be `0.2.1`. If last version was `0.1.3`, the breaking change should land in a version of `0.2.0`. Sorry for misleading you.

**Comment by ain** [comment]
In addition to the version, please also:
- update README with new options
- include distribution

**Comment by ain** [comment]
@beroes you didn't include distribution, [dist/iptools-jquery-modal.min.js](https://github.com/interactive-pioneers/iptools-jquery-modal/blob/master/dist/iptools-jquery-modal.min.js) is still old.

**Comment by beroes** [comment]
It is includet you can check the commit.

**Comment by ain** [comment]
Ah sorry, the diff was suppressed.

**Pull request merged**